components/ai_chat/core/browser/engine/engine_consumer.cc

```diff
@@ -54,11 +54,6 @@ bool EngineConsumer::CanPerformCompletionRequest(
     return false;
   }
 
-  const auto& last_turn = conversation_history.back();
-  if (last_turn->character_type != mojom::CharacterType::HUMAN) {
-    return false;
-  }
-
```

Comment:
A tool response will be submitted to the API when the last turn in the history will still be the assistant's.
This is low risk to remove since the reason for adding it, page content refining events, is no longer in the repo.